### ISSUE TITLE
ERROR: Script Debugger failed to connect, but being used anyway.  (Seems to be only for networking?)

### ISSUE THREAD

**Issue by realkotob** [godotengine/godot#32933]
**Godot version:** 3.2dev Latest master branch.

**OS/device including version:** Windows 10

**Issue description:**

When doing networking with rpc calls, the client is spammed with error messages after joining the game.

```
ERROR: Script Debugger failed to connect, but being used anyway.
   At: core/script_debugger_remote.cpp:140
```
**Steps to reproduce:**
Make the project throw errors in the rpc call across client-server.

**Minimal reproduction project:**
Standard rpc networking demo but make it throw errors in the rpc call.

**Comment by nikitavondel** [comment]
Experiencing this in my project too.

**Comment by nikitavondel** [comment]
For the people that develop with two godot projects open, I have found a solution:
Go to: Editor > Editor Settings > Network > Debug. And change the port.

One thing worth noting is that the editor settings seem to be shared globally across projects so if you change the port in one project and re-open another project with the Godot editor it will try to run on the newly set port too causing yet again the same issue.

**Comment by asheraryam** [comment]
@nikitavondel Thanks I'll use that workaround for now, it should be fine until the issue is fixed.

I wonder if it would be worth considering to move that setting to Project Settings instead?

**Comment by bitmammoth** [comment]
This is especially annoying on linux based projects since you have to run the executable from terminal to get the current debug without using a workaround. On windows systems you can see the debug still output to terminal by default but on linux not so much. I have tried to alter the flags for --remote-debug port settings but it seems to not respond.

**Comment by Faless** [comment]
The possibility of running multiple editors and debug with them has been implemented via #53241 (`4.0`) and #37067 (`3.x`).
Godot 4 also supports debugging multiple game instances using the same editor instance.

Closing as I'm also unable to reproduce the issue by forcing a debugger connection failure, but feel free to comment if you still encounter this error and I'll re-open.
